packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -61,7 +61,7 @@ class ThemeData {
     primarySwatch ??= Colors.blue;
     primaryColor ??= isDark ? Colors.grey[900] : primarySwatch[500];
     primaryColorBrightness ??= ThemeBrightness.dark;
-    accentColor ??= primarySwatch[500];
+    accentColor ??= isDark ? Colors.tealAccent[200] : primarySwatch[500];
```

Comment:
I thought this was what `accentColorBrightness` was about. You have to handle the case of it being bright and it being dark.